packages/components/src/slot-fill/test/__snapshots__/slot.js.snap

```diff
@@ -0,0 +1,74 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Slot should re-render Slot when not bubbling virtually 1`] = `
+Array [
+  <div>
+    1
+  </div>,
+  <button
+    onClick={[Function]}
+    type="button"
+  />,
+]
+`;
+
+exports[`Slot should re-render Slot when not bubbling virtually 2`] = `
+Array [
+  <div>
+    2
+  </div>,
+  <button
```

Comment:
Note that, previously, we weren't doing snapshot testing and the `button` component wasn't checked for.